helm-system-packages-brew.el

```diff
@@ -0,0 +1,214 @@
+;;; helm-system-packages-brew.el --- Helm UI for Mac OS ' homebrew. -*- lexical-binding: t -*-
```

Comment:
The OS name is officially "macOS".  Also no space before the apostrophe.

Reply:
Should be addressed by https://github.com/tuedachu/helm-system-packages/pull/1.